Dockerfile

```diff
@@ -0,0 +1,16 @@
+# We're using Python 3.10 running on the Alpine Linux distribution
+FROM python:3.10-alpine
+
+# Use this path (that contains the Dockerfile) as the working dir
+WORKDIR ./gerousiabot
+
+# Copy the required files into the docker image
+COPY ./gerousiabot ./gerousiabot
+COPY Pipfile* .
```

Comment:
actually this never worked for me cause i had the pipfile and the pipfile.lock 

got the message 


When using COPY with more than one source file, the destination must be a directory and end with a /

so you need to change it to `COPY Pipefile* ./`